mailbox/api/src/main/java/org/apache/james/mailbox/model/Quota.java

```diff
@@ -25,29 +25,29 @@
 
 public class Quota<T extends QuotaValue<T>> {
 
-    public static <T extends QuotaValue<T>> Quota<T> quota(T used, T max) {
```

Comment:
can you explain the rationals? Why do you think limit is a better naming than max?

Reply:
Of course : when there was a single max value, max is ok.
But then I introduce the "maxByScope" and max(a, b) returns the biggest value whereas limit(a, b) has no predefined math expectation, so I decided to use limitByScope.
For consistency, I changed max field to limit.